requirements/py3_dev.txt

```diff
@@ -1,3 +1,4 @@
 -r py3_common.txt
 -r dev.txt
 -r mypy.txt
+-r twisted.txt
```

Comment:
Regarding  requirement I agree with you it should be moved to dev.txt. 
Regarding section on the top of run-dev.py. When I launch this script on python3 it doesn't find twisted in environment(but twisted was installed in python2 environment, not in python3 env), so it means this section doesn't work and should be removed.